A32NX/html_ui/Pages/A32NX_Core/A32NX_GPWS.js

```diff
@@ -61,7 +61,7 @@ class A32NX_GPWS {
 
         const mda = SimVar.GetSimVarValue("L:AIRLINER_MINIMUM_DESCENT_ALTITUDE", "feet");
         const dh = SimVar.GetSimVarValue("L:AIRLINER_DECISION_HEIGHT", "feet");
-        const phase = SimVar.GetSimVarValue("L:AIRLINER_FLIGHT_PHASE", "Enum");
+        const phase = SimVar.GetSimVarValue("L:A32NX_FWC_FLIGHT_PHASE", "Enum");
```

Comment:
Are you sure this is the FWC flight phase? I'm having a hard time finding a reference for this.

Reply:
I've corrected this one as well